app/script/system_notification/SystemNotificationRepository.coffee

```diff
@@ -346,12 +346,22 @@ class z.SystemNotification.SystemNotificationRepository
 
     should_obfuscate = @_should_obfuscate_notification message_et
 
+    if z.util.Environment.electron and z.util.Environment.os.mac
+      icon = ''
+    else
+      try
+        icon = unless should_obfuscate then message_et.user().preview_picture_resource().generate_url()
```

Comment:
a reminder to myself: change to `should_obfuscate_sender` once #595 gets merged.